pkg/app/carbonapi/app.go

```diff
@@ -41,10 +41,11 @@ type App struct {
 	config       cfg.API
 	ZipperConfig cfg.Zipper
 
-	queryCache             cache.BytesCache
-	findCache              cache.BytesCache
-	TopLevelDomainCache    *expirecache.Cache
-	TopLevelDomainPrefixes []tldcache.TopLevelDomainPrefix
+	queryCache               cache.BytesCache
+	findCache                cache.BytesCache
+	TopLevelDomainCache      *expirecache.Cache
+	TopLevelDomainPrefixes   []tldcache.TopLevelDomainPrefix
+	NotFoundWhenTLDCacheMiss bool
```

Comment:
maybe NotFoundUponTLDCacheMiss? upon before noun, vs when before verf

Reply:
or maybe even better `ErrNotFoundUponTLDCacheMiss`

Reply:
I'll keep this in mind for the next times and refactor this. thx.